### DISCUSSION TITLE
Usability of clean_segment and clean_zone on iPhone and tablet

### DISCUSSION THREAD

**Discussion by ChrisHomeAssistant** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#326] [Q&A] [answered]
Dear Piotr,

first of all, thank you very much for all the work you put into this card. It is absolutely great!
I set up everything in my home assistant dashboard and all the features work fine (I'm a huge fan of the room selection via clean_segment...)

However, I'm always struggling when I use the card via iPhone or tablet. For example, I'm not able to adjust the rectangle after inserting one and I'm not able to select segments that are shown on the map.

Is there anything wrong in my configuration or is it a known issue?

Thanks
Christoph

**Comment by PiotrMachowski** [comment, marked as answer]
Is this the same behavior like [here](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/issues/230)? It should (probably) fix itself after changing mode back and forth.

**Reply by ChrisHomeAssistant** [reply]
Thank you very much for the quick answer. It seems to be similar. After switching back and forth it worked.

**Reply by PiotrMachowski** [reply]
Safari `¯\_(ツ)_/¯`
